config.md

```diff
@@ -2,6 +2,7 @@
 
 The container's top-level directory MUST contain a configuration file called `config.json`.
 The canonical schema is defined in this document, but there is a JSON Schema in [`schema/config-schema.json`](schema/config-schema.json) and Go bindings in [`specs-go/config.go`](specs-go/config.go).
+The media type for this file is configuration is `application/vnd.oci.runtime.config.v1+json`, and all specifications using that media type will require [JSON](glossary.md#json) objects with an [`ociVersion`](#specification-version).
```

Comment:
What are you suggesting?  I don't think that Markdown link will resolve.  Are you suggesting I drop the JSON and `ociVersion` requirements for that media type?  Do you expect those requirements to be overly restrictive?  How do you expect consumers to process instances of this media type if we break either restriction?